Creating database

anjelica-m opened the pull request:
Added database code to onCreate and fixed gradle files

anjelica-m committed:
Added Database to Project

Added the code to create the database collections onCreate, also updated gradle files to support firebase properly.

anjelica-m committed:
Removed some unused classes

Removed 2 classes I added thinking they would be used for checking the database

tjpollom committed:
Merge branch 'main' into Creating-Databases

tjpollom reviewed (approved):
Looks good, but in the future please try to stick only to files relevant to the problem being worked on. By that I mean specifically the Code and User classes, which are additional java files not used or referenced by the implementation of the database. Ideally, if something is done in addition to the point of the branch, they should have their own branch and merge request.

DanielleHopfe reviewed (changes requested):
The google-services.json file is missing when I run the project on my computer, this needs to be added to the project before we merge the database to main

sstpm committed:
Update .gitigore and add google-services.json

Remove the line in the project gitignore that prevents adding the
google-services.json file, which is required for firebase.

DanielleHopfe reviewed (approved):
now runs on my computer and all test cases appear properly on the database as far as I can tell
